packages/bpk-component-radio/examples.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import { action, BpkDarkExampleWrapper } from 'bpk-storybook-utils';
+
+import BpkRadio from './index';
+
+const loremIpsum = `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolorem dolores doloremque, expedita
+quaerat temporibus ipsam, ut, ipsa, velit sed assumenda suscipit dolore quod similique delectus numquam neque!
+Nesciunt, voluptate, illo.`;
+
+class GroupExample extends React.Component<{}, { value: string }> {
+  constructor() {
+    super();
+    this.state = { value: 'Apples' };
+  }
+
+  updateValue = value => {
+    action(`Radio changed. New value: ${value}`);
+    this.setState({ value });
+  };
+
+  render() {
+    const { value } = this.state;
+    const { ...rest } = this.props;
+    return (
+      <div>
+        {['Apples', 'Bananas', 'Strawberries', 'Quince'].map(fruit => (
```

Comment:
Should we make these examples more travel related if we end up using them on the docs site?